packages/http-server/src/server.ts

```diff
@@ -71,9 +71,9 @@ const replyHandler = <LoaderInput>(
           reply.headers(output.headers);
         }
 
-        reply.serializer((payload: unknown) => payload).send(output.body);
```

Comment:
This workaround was a fix for double encoded JSON, as fasting seems to assume it should JSON encode whatever you pass it. Is this no longer happening?

Reply:
Not exactly.

The reason this workaround was required was because the example generator was also taking care of the serialisation — so we had to bypass the one that the framework (Fastify) was offering.

Now that I separated the concepts (the serialisation is something the example generator should not care about) — we can remove the "do not serialise" part and let the framework do its job.